src/instance.ts

```diff
@@ -108,6 +206,18 @@ class Instance extends common.ServiceObject {
     this.requestStream = spanner.requestStream.bind(spanner);
     this.databases_ = new Map();
   }
+
+  createDatabase(name: string): Promise<CreateDatabaseResponse>;
```

Comment:
Lets delete this overload and mark `options` as optional on the other Promise returning one.